src/Donors/Endpoints/ListDonors.php

```diff
@@ -65,35 +78,153 @@ public function registerRoute()
                         'required' => false,
                         'validate_callback' => [$this, 'validateDate']
                     ],
+                    'sortColumn' => [
+                        'type' => 'string',
+                        'required' => false,
+                        'sanitize_callback' => 'sanitize_text_field',
+                    ],
+                    'sortDirection' => [
+                        'type' => 'string',
+                        'required' => false,
+                        'enum' => [
+                            'asc',
+                            'desc'
+                        ],
+                    ],
+                    'locale' => [
+                        'type' => 'string',
+                        'required' => false,
+                        'default' => get_locale(),
+                    ],
                 ],
             ]
         );
     }
 
     /**
-     * @param WP_REST_Request $request
+     * @unreleased Change this to use the new ListTable class
      * @since 2.20.0
      *
+     * @param WP_REST_Request $request
+     *
      * @return WP_REST_Response
      */
     public function handleRequest(WP_REST_Request $request): WP_REST_Response
```

Comment:
Can you please write a feature unit test for this? I can help you set it up but I think it would be very valuable to have a stable test for all this logic.

Reply:
Resolved: c5842b2f28ee31da5602c0a897f1d66f84d7b01d